Park/Host/Program.cs

```diff
@@ -0,0 +1,195 @@
+                    {
+                        case 1:
+                            GetFullList(l, p, move);
+                            break;
+                        case 2:
+                            GiveFood(l, p, move);
+                            break;
+                        case 3:
+                            MakePregnant(l, p, move);
+                            break;
+                        case 4:
+                            EndMove(l, p, move);
+                            break;
+                        case 5:
+                            Console.WriteLine("Будьте уверены, как только вы покинете игру, все ваши питомцы умрут с голоду. Потому что никому, кроме вас, они нужны.");
+                            Console.WriteLine("Введите Y, чтобы выйти.");
+                            string? exit = Console.ReadLine();
+                            switch (exit)
+                            {
+                                case "Y":
+                                    return;
+                                default:
+                                    Menu(l, p, move);
+                                    break;
+                            }
+                            break;
+                        default:
+                            Console.WriteLine("Что-то пошло не так");
+                            Menu(l, p, move);
+                            break;
+                    }
+                }
+                else
+                {
+                    Console.WriteLine("Неверное число\n");
+                    continue;
+                }
+            }
+
+            static void GetFullList(List<Animal> l, Plant[] p, int move)
+            {
+                Console.WriteLine("\n------------------------------------------------------");
+                Console.WriteLine("Список животных:\n");
+                foreach (var a in l)
+                {
+                    Console.WriteLine($"| Индекс: {l.IndexOf(a)}, Имя: {a?.name}, Размер: {a.size}, Возраст: {a.age}, Мама: {a?.femaleParent?.name}, Папа: {a?.maleParent?.name}, Съедобное растение: {a?.allowedPlant?.name}");
+                }
+                Console.WriteLine("Список растений:\n");
+                for (int i = 0; i < p.Length; i++)
+                {
+                    Console.WriteLine($"| Индекс: {i}, Название: {p[i]?.name}, Размер: {p[i]?.size}");
+                }
+                Console.WriteLine("------------------------------------------------------\n");
+                Menu(l, p, move);
+            }
+
+            static void GiveFood(List<Animal> l, Plant[] p, int move)
+            {
+                Console.WriteLine("Чтобы накормить животное, необходимо выбрать индекс особи и индекс растения");
+                while (true)
+                {
+                    Console.WriteLine("Введите индекс животного: ");
+                    string aChoice = Console.ReadLine();
+                    int.TryParse(aChoice, out int x);
```

Comment:
если ввод не парсится к int, то...
x будет инициализирован значением по умолчанию - нулем.
не знаю ожидаемое это поведение для вас или нет.
имхо, лучше падать:
```suggestion
                    x = int.Parse(aChoice);
```